LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2021, Salesforce.com, Inc.
```

Comment:
I didn't do this. I think it happened as a part of a git commit hook?

Reply:
This was done by `dev-scripts` 😃 
https://github.com/forcedotcom/dev-scripts/pull/35